src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -38,15 +39,74 @@
         </resultlist>')
   end
 
-  describe 'submit_request' do
+  describe "POST #submit_request" do
+    RSpec.shared_examples "a response of a successfull submit request" do
```

Comment:
On 2016-08-19 02:27:08 -0700, Björn Geuken wrote:
> > @@ -38,15 +39,74 @@
> >          </resultlist>')
> >    end
> >  
> > -  describe 'submit_request' do
> > +  describe "POST #submit_request" do
> > +    RSpec.shared_examples "a response of a successfull submit request" do
> 
> Fixed the typo. Thanks!
> 
Nit-pick: the commit message should be "successfull -> successful":)
(commit 96a924a3e41014226d51d2420d0bbaed6c6d1f56)